docs/release-notes.txt

```diff
@@ -4,6 +4,82 @@
  Release Notes
 ###############
 
+**************
+ Version 0.14
+**************
+
+Version 0.14.0
+==============
+
+**Release Date:** February 4, 2021
+
+**New Features**
+
+-  WebUI: Revamp cluster page with information about configured resource
+   pools.
+
+**Removed Features**
+
+-  Trial API: Remove the old PyTorch APIs, including:
+
+   -  the ``build_model``, ``optimizer``, and ``create_lr_scheduler``
+      methods in :class:`~determined.pytorch.PyTorchTrial`;
+
+   -  the callback ``on_before_optimizer_step``;
+
+   -  the field ``optimizations.mixed_precision`` in the experiment
+      configuration;
+
+   -  the ``model`` arguments to
+      :meth:`~determined.pytorch.PyTorchTrial.train_batch`,
+      :meth:`~determined.pytorch.PyTorchTrial.evaluate_batch`, and
+      :meth:`~determined.pytorch.PyTorchTrial.evaluate_full_dataset`.
+
+   Model code that uses these APIs will no longer run in Determined
+   0.14.0 or later. However, model checkpoints produced by old
+   experiments that used these APIs will still be supported.
+
+**Improvements**
+
+-  Documentation: Add a topic guide on :ref:`debugging models
+   <model-debug>`. The new guide will walk you step-by-step through
+   solving problems with a model in Determined, with a focus on testing
+   features incrementally until the model is fully working. It may also
+   be useful when porting new models to Determined.
+
+-  Documentation: Add a topic guide on :ref:`commands and shells
+   <commands-and-shells>`. It describes how to use Determined's support
+   for managing GPU-powered batch commands and interactive shells.
+
+-  **Breaking Change** Metrics: The trial and checkpoint API endpoints
+   can now return non-scalar metric values, which are represented as
+   JSON objects or `protobuf structs
+   <https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Struct>`__.
+
+-  PyTorch: Upgrade the default PyTorch version to 1.7.0.
+
+-  TensorFlow: Upgrade the default TensorFlow versions to 1.15.5 and
+   2.4.1.
+
+-  REST API: Improve the performance of the experiments API.
+
+**Bug Fixes**
+
+-  API: Fix an issue where requesting checkpoint or trial details of a
+   trial that had non-scalar metric values associated with it would
+   fail.
+
+-  Database: Migrate ``public.trial_logs.id`` to be an ``int8`` in
+   Postgres, instead of an ``int4``. This avoids issues for customers
+   with extremely large amounts of trial logs. **Note**: This migration
+   may be more time-consuming than usual for deployments with large
```

Comment:
blocking: Probably "will" be more time-consuming, right?

Reply:
I don't think we know that for sure in all of these cases, so I think "will" is a bit strong.

Reply:
I changed my mind, will be in next iteration.